app/assets/javascripts/components/sidebar/sidebar.es6.jsx

```diff
@@ -27,7 +27,7 @@ class Sidebar extends Component {
         transition: 'left 0.375s ease-out',
       },
       hidden: {
-        left: '-172px',
+        left: '-200px',
```

Comment:
Turns out if you zoom out a lot the sidebar will appear again, so we'll think of a better way to do this.